test/server/cards/02_SHD/units/GentleGiant.spec.ts

```diff
@@ -0,0 +1,60 @@
+describe('Gentle Giant', function() {
+    integration(function(contextRef) {
+        describe('Gentle Giant\'s ability', function() {
+            beforeEach(function () {
+                contextRef.setupTest({
+                    phase: 'action',
+                    player1: {
+                        groundArena: [
+                            { card: 'gentle-giant', damage: 1 },
+                            { card: 'r2d2#ignoring-protocol', damage: 3 },
+                            { card: 'c3po#protocol-droid', damage: 1 }],
+                    },
+                    player2: {
+                        groundArena: [{ card: 'wampa', damage: 1 }]
```

Comment:
Let's remove the damage from anything that isn't actually going to be targeted for the heal effect just to make it super obvious what the test will be doing. That way if anyone accidentally breaks this test and needs to see what's going on they can tell really fast.